tests/objax2tf.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unittests for Objax2Tf converter."""
+
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+import objax
+from objax.zoo.wide_resnet import WideResNet
+import tensorflow as tf
+
+
+BATCH_SIZE = 4
+NCHANNELS = 3
+NCLASSES = 10
+IMAGE_SIZE = 32
+
+
+class TestObjax2Tf(unittest.TestCase):
+
+    def verify_converted_predict_op(self, objax_op, tf_op, shape):
+        x1 = np.random.normal(size=shape)
+        x2 = np.random.normal(size=shape)
+        # due to differences in op implementations, there might be small numerical
+        # differences between TF and Objax, thus comparing up to 1e-4 relative tolerance
+        np.testing.assert_allclose(objax_op(x1), tf_op(tf.convert_to_tensor(x1, dtype=tf.float32)), rtol=1e-4)
+        np.testing.assert_allclose(objax_op(x2), tf_op(tf.convert_to_tensor(x2, dtype=tf.float32)), rtol=1e-4)
+
+    def test_convert_wrn(self):
+        # Make a model
+        model = WideResNet(NCHANNELS, NCLASSES, depth=4, width=1)
+        model_vars = model.vars()
+        # Prediction op without JIT
+        predict_op = lambda x: objax.functional.softmax(model(x, training=False))
+        predict_tf = objax.util.Objax2Tf(objax.Function(predict_op, model_vars))
+        # Compare results
+        self.verify_converted_predict_op(predict_op, predict_tf,
+                                         shape=(BATCH_SIZE, NCHANNELS, IMAGE_SIZE, IMAGE_SIZE))
+        # Predict op with JIT
+        predict_op_jit = objax.Jit(predict_op, model_vars)
```

Comment:
```suggestion
        predict_op = objax.nn.Sequential([objax.ForceArgs(model, training=False), objax.functional.softmax])
        predict_tf = objax.util.Objax2Tf(predict_op)
        # Compare results
        self.verify_converted_predict_op(predict_op, predict_tf,
                                         shape=(BATCH_SIZE, NCHANNELS, IMAGE_SIZE, IMAGE_SIZE))
        # Predict op with JIT
        predict_op_jit = objax.Jit(predict_op)
```

Reply:
What is the reason for this change?

Reply:
I'm trying to move away from functions as arguments: https://github.com/google/objax/issues/168
Plus it's the features of Objax more efficiently, and no need to pass vars to jit etc...